### ISSUE TITLE
Missing Characters

### ISSUE THREAD

**Issue by mahiuchun** [siuying/cantonese-syllables#2]
Common Cantonese character 嘅, 咗 are not available from Multi-function Chinese Character Database and thus the scraped data.

How about try alternative data sources, for example:
http://www.iso10646hk.net/jp/
http://unicode.org/reports/tr38/#kCantonese
https://raw.githubusercontent.com/lukhnos/openvanilla/master/DataTables/jyutping.cin

I guess there is no absolute authoritative source of Cantonese readings. But we'd better know to what extent these sources agree with each other.

**Comment by siuying** [comment]
Good point. the database from http://www.iso10646hk.net might worth some investigation. But they release the data as a huge PDF :confused:

**Comment by mahiuchun** [comment]
Hi, another aspect is finding out the ultimate data source, the Cantonese dictionaries.
According to articles like:
http://cantoneseculture.com/page_MediaInternetJournal/mediaIntJrl20090415.aspx
http://web.hku.hk/~cmsi/item2/item2_3/item2_3_3.pdf

Currently we have at least the following Cantonese dictionaries:
1. 粵音韻彙 by 黃錫凌
2. 廣州音字典 by 饒秉才
3. 常用字廣州話讀音表 by 香港教育署
4. 粵音正讀字彙 by 何文匯 et al.
5. 廣州話正音字典 by 詹伯慧 et al.

And the canonical readings of various characters are controversial; 從切 or 從眾 is never settled.

The modern, open-data way of help current situation should be making the process of 審音 more transparent and public accessible.
If we have the readings of dictionaries in machine readable form, it is trivial to find out where they disagrees. And the public can comment on controversial characters.
The traditional way of experts do the 審音 themselves simple create a situation like https://xkcd.com/927/

CUHK has web page for 粵音韻彙 also: http://humanum.arts.cuhk.edu.hk/Lexis/Canton/
Not sure about other dictionaries yet.

**Comment by ghost** [comment]
Hi, guys. I would say grabbing pronunciations from _香港電腦用漢字粵語拼音表_ is a stupid idea. The editors of that book reasoned out the pronunciations of many rarely used characters **by analogy**, even for non-Cantonese or non-Chinese characters. And the data is out of date (Sep. 2004) compared to their online searching tool (last updated in 2011). I've [transformed the PDF into plain text](https://github.com/jyutping/rime-schema-jyutping/blob/8afa70c8c1a808eaaeac5d6e935abc5c79334349/refs/jyut-jam-table.txt) (quite precise) and scraped their website. My conclusion is: It is not worth importing their data. (Yeah, I've done this stupid thing for you.)

從切 is most reliable when we are looking up the rarely used characters, and is not good for determining the standard sounds for this modern world. I want to recommend this book to you: http://www.savepropercantonese.com

Welcome to join us if you'd like to share more open-data. :book: 
https://github.com/jyutping

**Comment by siuying** [comment]
>   I've transformed the PDF into plain text (quite precise) and scraped their website. My conclusion is: It is not worth importing their data. 

Wow, this is an impressive feat by itself ;-)
